.travis.yml

```diff
@@ -1,15 +1,22 @@
----
 sudo: false
 language: ruby
-
 rvm:
 - 2.0.0
 - 2.1.5
 - 2.2.0
 - 2.3.0
-
 script:
 - cd service_management/azure && bundle install && bundle exec rake test:unit
 - if [ "$INTEG_RECORDED" == "true" ] ; then bundle exec rake test:recorded ; fi
 - cd ../../resource_management && bundle install
-- if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ] ; then rake test:integration ; fi
\ No newline at end of file
+- if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ] ; then rake test:integration ; fi
+deploy:
+  provider: rubygems
+  api_key:
+    secure: h1O/CbvwR2fsS013v7QAlW8qLIXrMDED9+HPDi5fM5o/Q/PU7uJTRI3Aold76xzXJj0eDloNIXlocdiYaJ3fh4u4UYv1HjTIW5yz9L30fjhuuWw7lMvBUHz0eYUTPWwWEZvp2aJY07C5KVRn+0sjZs9aKTUUIfFtHsW3uSZZpfY=
+  gemspec: service_management/azure/azure.gemspec
+  gem:
+    master: azure
+  on:
+    tags: true
+    repo: Azure/azure-sdk-for-ruby
```

Comment:
I think it's optional but I used `travis setup rubygems` command to set it up and it asked me to specify.